models/us_cms_open_payments/code/upload.py

```
@@ -0,0 +1,112 @@
+    storage = bd.Storage(dataset_id=c.GCP_DATASET_ID, table_id=table)
+    storage.delete_table(mode="staging", not_found_ok=True)
+
+    bd.Table(dataset_id=c.GCP_DATASET_ID, table_id=table).create(
+        path=str(path),
+        source_format="parquet",
+        if_table_exists="replace",
+        if_storage_data_exists="replace",
+        if_dataset_exists="pass",
+    )
+
+    query = f"select count(*) as n from `{BILLING_PROJECT}.{c.GCP_DATASET_ID}_staging.{table}`"
```

Ruff (0.16.1)

[error] Possible SQL injection vector through string-based query construction

(S608)